src/ts/features/block-quotes.ts

```diff
@@ -0,0 +1,51 @@
+import {Feature, Settings} from '../utils/settings'
+
+export const config: Feature = {
+    id: 'blockquotes',
+    name: 'Blockquote Support',
+    settings: [{type: 'string', id: 'blockquote_prefix', initValue: '> ', label: 'Blockquote Prefix'}],
+    description:
+        'If you start a block with these characters, custom styles will be added to show the block as a blockquote.',
+}
+
+async function blockquotes() {
+    const blocks = document.querySelectorAll('div.roam-block')
+    for (const block of blocks as NodeListOf<Element>) {
```

Comment:
this is more relevant if we stay with the time-based rendering and when we add more renderings but: it probably makes sense to have a function that will get all the blocks and then apply all transformations we have to each of them vs having each transformation fetch the blocks.